core/src/main/scala/chisel3/internal/firrtl/IR.scala

```diff
@@ -186,6 +186,15 @@ private[chisel3] object ir {
     }
   }
 
+  /** Property expressions.
+    *
+    * Property expressions are conceptually similar to Nodes, but only exist as a tree of Args in-memory.
+    */
+  case class PropExpr(sourceInfo: SourceInfo, tpe: firrtlir.PropertyType, op: firrtlir.PropPrimOp, args: Arg*)
```

Comment:
```suggestion
  case class PropExpr(sourceInfo: SourceInfo, tpe: firrtlir.PropertyType, op: firrtlir.PropPrimOp, args: List[Arg])
```
Don't use varargs in classes, just pick the container type. I'm guessing this is only ever 1-2 so List is a good choice.

Reply:
Sounds good. I was copying this from DefPrim, but I can see why its better to not use varargs.